README.rst

```diff
@@ -320,8 +320,22 @@ You can further `customize your plugin`_ as other plugins.
 Adding plugins to the "CMS Plugins" dropdown
 --------------------------------------------
 
-If you have another plugin that you want to use inside texts you can make them appear in the dropdown by making them text_enabled.
-Check in `django-cms doc`_ how to do this.
+If you have created a plugin that you want to use inside texts you can make them appear in the dropdown by 
+making them `text_enabled`. This means that you assign the property `text_enabled` of a plugin to `True`, 
+the default value is `False`. Here is a very simple implementation::
+
+    class MyTextPlugin(TextPlugin):
+        name = _(u"My text plugin")
+        model = MyTextModel
+        text_enabled = True
+
+When the plugin is picked up, it will be available in the "CMS Plugins" dropdown, which you can find in the editor. 
```

Comment:
Please use emphasis/italics for names of interface elements (except buttons, which should be strong/bold):
```
... *CMS Plugins*